AutoT4MVC/AutoT4MVC.csproj

```diff
@@ -38,7 +39,7 @@
     <AssemblyName>AutoT4MVC</AssemblyName>
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>Key.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Okay. We'll leave it as is. It works on my machine in VS 2012, so I guess people still using that will need to install .NET 4.6 or just not upgrade to the latest. I'll get this released today.

Thanks again mate 👍